tests/test_shortcuts.py

```diff
@@ -12,8 +12,12 @@
 class TestGetCurrentApp(object):
 
     @pytest.fixture
-    def token(self):
-        return mixer.blend(AccessToken)
+    def application(self):
+        return mixer.blend(Application)
+
+    @pytest.fixture
+    def token(self, application):
+        return mixer.blend(AccessToken, application=application)
```

Comment:
This `AcessToken` still uses `from oauth2_provider.models import AccessToken`. Could you please change it to use from `compat` module? Other modules may need a double check too.